airflow/executors/celery_executor.py

```diff
@@ -78,6 +80,45 @@ def execute_command(command_to_exec: CommandType) -> None:
     """Executes command."""
     BaseExecutor.validate_command(command_to_exec)
     log.info("Executing command in Celery: %s", command_to_exec)
+
+    if settings.EXECUTE_TASKS_NEW_PYTHON_INTERPRETER:
+        _execute_in_subprocees(command_to_exec)
+    else:
+        _execute_in_fork(command_to_exec)
+
+
+def _execute_in_fork(command_to_exec: CommandType) -> None:
+    pid = os.fork()
```

Comment:
Do we need to fork it? Shouldn't we just execute it in current process (celery worker process)?

Reply:
Can't cos of the `logging.shutdown()` at the end of task_run (which we need to keep, as that's when remote logs are uploaded. https://github.com/apache/airflow/pull/11327#issuecomment-705139313